.github/workflows/tests.yml

```diff
@@ -22,18 +24,16 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2
         with:
-          node-version: 6
+          node-version: 14
           cache: 'npm'
-      - run: npm install --no-audit && npm install --no-audit karma-cli karma-sauce-launcher
-      - name: Setup Sauce Labs
-        run: sh -e /etc/init.d/xvfb start
-        env:
-          DISPLAY: ':99.0'
-      - run: npm test
-      - run: ./node_modules/.bin/karma start karma.conf-sauce.js
+      - name: Install dependencies
+        run: |
+          npm install --no-audit
```

Comment:
We want `npm ci` here, since we had package lock

Reply:
Thanks. I will have a look at the documentation, because I'm not really familiar with the `ci` subcommand.